x-pack/test/api_integration/apis/ml/modules/setup_module.ts

```diff
@@ -197,6 +240,36 @@ export default ({ getService }: FtrProviderContext) => {
             await ml.api.waitForJobState(job.jobId, job.jobState);
             await ml.api.waitForDatafeedState(datafeedId, job.datafeedState);
           }
+
+          // compare model memory limits for created jobs
+          const expectedModelMemoryLimits = testData.expected.jobs
+            .map(j => ({
+              id: j.jobId,
+              modelMemoryLimit: j.modelMemoryLimit,
+            }))
+            .sort(compareById);
+
+          const {
+            body: { jobs },
+          }: {
+            body: {
+              jobs: Job[];
+            };
```

Comment:
would be better to provide a return type for `getAnomalyDetectionJob` method

Reply:
I agree, but would touch other files which use it, so should be done in a refactor follow up.